Updating Tutorials for 0.3.0 (#86)
acquire-project/acquire-docs: docs/tutorials/setup_acquisition/configure.md

# (x, y) size of the image in pixels
config.video[0].camera.settings.shape = (1024, 768)

# Specify the pixel type as uint16
config.video[0].camera.settings.pixel_type = acquire.SampleType.U16
```

## Configure `Storage`
`Storage` objects have 2 attributes, `settings`, a `StorageProperties` object, and an optional attribute `identifier`, which is an instance of the `DeviceIdentifier` class described above.

> `StorageProperties` has 2 attributes `external_metadata_json` and `filename` which are strings of the filename or filetree of the output metadata in JSON format and image data in whatever format corresponds to the selected storage device, respectively. `first_frame_id` is an integer ID that corresponds to the first frame of the current acquisition and is typically 0. `pixel_scale_um` is the camera pixel size in microns. `acquisition_dimensions` is a list of instances of the `StorageDimension` class, one for each acquisition dimension with the fastest changing dimension listed first and the append dimension listed last. For more information on using the `StorageDimension` class, check out [Chunking Data for Zarr Storage](../zarr/chunked.md). `enable_multiscale` is a boolean used to specify if the data should be saved as an image pyramid. See the [Multiscale tutorial](../zarr/multiscale.md) for more information.

Reviewer: ```suggestion
`StorageProperties` has 2 attributes `external_metadata_json` and `filename` which are strings of the filename or filetree of the output metadata in JSON format and image data in whatever format corresponds to the selected storage device, respectively. `first_frame_id` is an integer ID that corresponds to the first frame of the current acquisition and is typically 0. `pixel_scale_um` is the camera pixel size in microns. `acquisition_dimensions` is a list of `StorageDimension`, one for each acquisition dimension, ordered from fastest changing to slowest changing. For more information on using the `StorageDimension` class, check out [Chunking Data for Zarr Storage](../zarr/chunked.md). `enable_multiscale` is a boolean used to specify if the data should be saved as an image pyramid. See the [Multiscale tutorial](../zarr/multiscale.md) for more information.
```
